lib/validation.js

```diff
@@ -18,10 +18,11 @@
  *
  * @module numberValidation
  */
+const isInt = require('validator/lib/isInt');
 
-module.exports = function numberValidation(input, settings) {
-  if (input.target.value === '' && !settings.target.empty) {
-    return `${input.target.name} cannot be left blank!`;
+module.exports = function numberValidation(input) {
+  if (input.target.value && !isInt(input.target.value.toString())) {
```

Comment:
The Number input type can contain either ints or floats. You should check for both `isInt` and `isDecimal`. `isNumeric` is also probably a good first thing to throw as well, but probably not needed